app/abilities/general_ability.rb

```diff
@@ -0,0 +1,15 @@
+class GeneralAbility
+  include CanCan::Ability
+
+  def initialize(user)
+    if alpha?(user)
+      can :manage, :mailchimp
+    end
+  end
+  
+  private
+
+  def alpha?(user)
+    user.account_tester_level == 'alpha'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.